src/base/reset.scss

```diff
@@ -0,0 +1,22 @@
+@import "../config/index.scss";
```

Comment:
Does this file need to import the config?

Reply:
not currently, could potentially in the future if htere ever is a specific value needed, unless it's intended purpose will always be to set things to "0".. but just went with it for consistency.

Reply:
I'm fine with cutting here

Reply:
I guess it's a matter of whether this file (or typography) are being imported directly into the react components? Not sure how these more general base styles are being handled currently, or if they are just relying on Classic for these things. If they are being pulled in in some way, then including it for consistency makes sense...

Reply:
I could see a scenario where typography is brought in by itself for various projects... reset less so.  Adding that config in the base files is more about future projects like the marketing site

Reply:
Alrighty, well I'm fine either way. Take it or leave it.